lib/devise/orm/active_graph/counter_increment.rb

```diff
@@ -0,0 +1,9 @@
+module CounterIncrement
+	def increment_counter(property, id)
+		obj = find(id)
+		counter = obj.send(property) || 0
+		obj.update_attribute(property, counter + 1)
+	end
+end
+
+ActiveGraph::Node::ClassMethods.send :include, CounterIncrement
```

Comment:
why not
```suggestion
ActiveGraph::Node.extend CounterIncrement
```
? 
And. generally this should be done in the main entry file.